src/app/clusters/network-commissioning/network-commissioning-events.cpp

```diff
@@ -0,0 +1,156 @@
+#include "network-commissioning-events.h"
```

Comment:
This file isn't about events, afaict.  It's about command responses.  What am I missing?

Reply:
Merged this file into the main handler file now.